source/catalog.html

```diff
@@ -0,0 +1,245 @@
+          <dl class="addition-card__feature">
+            <dt class="addition-card__description">5 литров</dt>
+            <dd class="addition-card__value">50 Р.</dd>
+          </dl>
+          <a class="addition-card__order-button button" href="#">заказать</a>
+        </div>
+
+        <div class="add-goods__card addition-card">
+          <p class="addition-card__title card-title">молоко</p>
+          <dl class="addition-card__feature">
+            <dt class="addition-card__description">1 литр</dt>
+            <dd class="addition-card__value">100 Р.</dd>
+          </dl>
+          <a class="addition-card__order-button button" href="#">заказать</a>
+        </div>
+
+        <div class="add-goods__card addition-card">
+          <p class="addition-card__title card-title">витамины</p>
+          <dl class="addition-card__feature">
+            <dt class="addition-card__description">1 упаковка (30 г)</dt>
+            <dd class="addition-card__value">300 Р.</dd>
+          </dl>
+          <a class="addition-card__order-button button" href="#">заказать</a>
+        </div>
+      </section>
+
+      <p class="catalog-promo">Закажите все и получите чехол для кота в подарок!</p>
+    </main>
+
+    <footer class="page-footer">
+      <div class="page-footer__wrapper">
+        <section class="page-footer__contacts contacts">
+          <h2 class="visually-hidden">Контакты</h2>
+
+          <p class="contacts__call">приглашаем к сотрудничеству дилеров!</p>
+          <p class="contacts__adress">ул. Большая Конюшенная, д. 19/8 Санкт-Петербург</p>
+          <div class="contacts__map"></div>
+        </section>
+
+        <a class="page-footer__logo-name">
+          <img src="img/logo-name.svg" width="101" height="18" alt="Cat Energy">
+        </a>
+
+        <section class="page-footer__social social">
+          <h2 class="visually-hidden">Ссылки на соцсети разработчика.</h2>
+
+          <ul class="social__list">
+            <li class="social__item">
+              <a class="social__link social__link--vkontakte" href="https://vk.com/htmlacademy">Мы в Вконтакте</a>
+            </li>
+            <li class="social__item">
+              <a class="social__link social__link--youtube" href="https://www.youtube.com/user/htmlacademyru">Мы в Ютубе</a>
+            </li>
+            <li class="social__item">
+              <a class="social__link social__link--telegram" href="https://t.me/htmlacademy">Мы в Телеграме</a>
+            </li>
+          </ul>
+        </section>
+
+        <section class="page-footer__copyright copyright">
+          <h2 class="visually-hidden">Ссылка на сайт разработчика.</h2>
+
+          <a class="copyright__link copyright__link--name" href="https://htmlacademy.ru/intensive/adaptive">
+            <span>HTML Academy</span>
+            <span class="visually-hidden">Логотип HTML Academy.</span>
+            <img src="img/html academy.svg" width="27" height="34" alt="HTML Academy">
```

Comment:
```suggestion
            <img src="img/html-academy.svg" width="27" height="34" alt="HTML Academy">
```
И файл `html academy.svg` нужно переименовать в `html-academy.svg`.

Reply:
Исправлено